File: app/src/main/java/com/comeze/rangelti/consultacnpj/views/custom/CustomApplication.java
package com.comeze.rangelti.consultacnpj.views.custom;

import android.app.Application;

import com.comeze.rangelti.consultacnpj.R;

import uk.co.chrisjenx.calligraphy.CalligraphyConfig;

public class CustomApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
                .setDefaultFontPath("RobotoSlab-Regular.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build());
    }
}

File: app/src/main/java/com/comeze/rangelti/consultacnpj/views/custom/CustomApplication.kt
package com.comeze.rangelti.consultacnpj.views.custom

import android.app.Application

import com.comeze.rangelti.consultacnpj.R

import uk.co.chrisjenx.calligraphy.CalligraphyConfig

class CustomApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        CalligraphyConfig.initDefault(CalligraphyConfig.Builder()
                .setDefaultFontPath("RobotoSlab-Regular.ttf")
                .setFontAttrId(R.attr.fontPath)
                .build())
    }
}